release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -475,6 +475,61 @@ For more information, see xref:../registry/configuring_registry_storage/configur
 [id="ocp-4-17-storage_{context}"]
 === Storage
 
+[id="ocp-4-17-storage-efs-vol-metrics"]
+==== AWS EFS storage CSI usage metrics is generally available
```

Comment:
The title refers to this feature singularly, but the intro sentence refers to it in the plural. Since there are presumably more than one metric, it might be best to just change the title to refer to the metrics (plural) rather than the metrics feature (singular).

```suggestion
==== AWS EFS storage CSI usage metrics are generally available
```